agent/engine/docker_task_engine_test.go

```diff
@@ -156,6 +156,7 @@ func TestBatchContainerHappyPath(t *testing.T) {
 	default:
 	}
 	eventsReported.Wait()
+	client.EXPECT().DescribeContainer(gomock.Any()).AnyTimes()
```

Comment:
If the 'task'  get transited  to "steady state" before timeout, then 'DescribeContainer()' is NOT called at all.  In that case, Times(1) will fail the test.  
p.s on my ubuntu platform,   the 'task' always get to 'steady state' before timeout.